### ISSUE TITLE
[Bug] Missing replies for the last posted message

### ISSUE THREAD

**Issue by Antreesy** [nextcloud/spreed#10054]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

## Steps to reproduce
1. Write a reply to the message
2. Wait until conversations will be fetched again
3. See results

### Expected behaviour
Parent id should persist in the message object from store

### Actual behaviour
When we're fetching conversations, this parameter is missing for the lastMessage:
```
roomToken: {
    ...
    lastMessage: {
        ...
        parent: 'parent_id'
    }
}
```
After we patch conversations, messageStore is also updated with lastMessage (for some reason), removing this field from store (because it doesn't exist in the last response)

### Possible solutions
1. Pass the `parent` parameter when fetch conversations :warning: (see below)
2. Or fix it at messagesStore :green_circle: 

## Talk app

**Talk app version:** master / 17

**Custom Signaling server configured:** yes

**Comment by nickvergessen** [comment]
We can't really load the parent information due to the amount of requests that would trigger. Can we instead adjust the updating to not remove a parent if we are updating from the conversation list?

**Comment by Antreesy** [comment]
Fixed with #10058
